specification/healthbot/resource-manager/Microsoft.HealthBot/stable/2021-06-10/healthbot.json

```diff
@@ -0,0 +1,791 @@
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.HealthBot/healthBots": {
+      "get": {
+        "description": "Returns all the resources of a particular type belonging to a subscription.",
+        "operationId": "Bots_List",
+        "x-ms-examples": {
+          "List Bots by Subscription": {
+            "$ref": "./examples/ListBotsBySubscription.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The resource provider should return 200 (OK) to indicate that the operation completed successfully. For other errors (e.g. internal errors) use the appropriate HTTP error code.\nThe nextLink field is expected to point to the URL the client should use to fetch the next page (per server side paging). This matches the OData guidelines for paged responses. If a resource provider does not support paging, it should return the same body but leave nextLink empty for future compatibility.\nFor a detailed explanation of each field in the response body, please refer to the request body description in the PUT resource section. ",
+            "schema": {
+              "$ref": "#/definitions/BotResponseList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.HealthBot/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "description": "Lists all the available Azure Health Bot operations.",
+        "operationId": "Operations_List",
+        "x-ms-examples": {
+          "Get Operations": {
+            "$ref": "./examples/GetOperations.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/AvailableOperations"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Identity": {
```

Comment:
>Identity" [](http://example.com/codeflow?start=5&length=9)

Where is this definition referred? I would recommend to move such definition which is applicable across RP's to [https://github.com/Azure/azure-rest-api-specs/blob/bf44a3b7e5817ffc57b205a6d63c283af96d222b/specification/common-types/resource-management/v2/types.json](https://github.com/Azure/azure-rest-api-specs/blob/bf44a3b7e5817ffc57b205a6d63c283af96d222b/specification/common-types/resource-management/v2/types.json)

Reply:
@chiragg4u I referenced it from [here](https://github.com/Azure/azure-rest-api-specs/blob/3a2c0bce80c4572eea98a031dc6dbd5b26dc7f99/specification/servicefabric/resource-manager/Microsoft.ServiceFabric/stable/2020-03-01/application.json#L1446) to overcome an issue with one of the validations (related to avoiding anonymous types).
Should I create a new PR for updating the "common-types" file, or should I do it in this PR?